engine/class_modules/priest/sc_priest.cpp

```diff
@@ -538,6 +543,10 @@ struct smite_t final : public priest_spell_t
       sim->print_debug( "{} adjusted cooldown of Chastise, by {}, without Apotheosis.", priest(),
                         cooldown_base_reduction );
     }
+    if ( priest().talents.discipline.harsh_discipline.enabled() )
+    {
+      priest().buffs.harsh_discipline->increment();
```

Comment:
I would use `trigger()` here instead, functionally they are the same in this case.